scout/ecm_prep.py

```diff
@@ -6547,7 +6553,7 @@ def gen_tsv_facts(
             updated_tsv_fracs, updated_tsv_shapes = self.apply_tsv(
                 load_fact, ash_czone_wts, eplus_bldg_wts, cost_fact_hourly,
                 carbon_fact_hourly, mskeys, bldg_sect, eu, opts, cost_yr_map,
-                carb_yr_map)
+                carb_yr_map, save_shp_warn)
```

Comment:
Similar comment to above, this looks like it will always be passed as an empty list.